tq42/experiment_run.py

```diff
@@ -70,6 +71,52 @@ def __repr__(self) -> str:
     def __str__(self) -> str:
         return f"ExperimentRun: {MessageToJson(self.data, preserving_proto_field_name=True)}"
 
+    @property
+    def completed(self) -> bool:
+        """
+        Check if the experiment run is completed
+
+        :returns: true if the experiment run is in the state `COMPLETED`
+        """
+
+        return self.data.status == ExperimentRunStatusProto.COMPLETED
+
+    @property
+    def result(self) -> Optional[dict[str, Any]]:
```

Comment:
if we have these properties should we also have `inputs` and `parameters` as properties? 🤔 Just curious what you think about that.

Reply:
Is there a use case where the user would want to extract those fields? They probably have this information anyway.

Reply:
Good question, I am not sure tbh but you might be right, let's just implement this if we see the use case sometime